src/pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -141,7 +141,7 @@ def _validate_condition_metric(self, logs: Dict[str, Tensor]) -> bool:
         monitor_val = logs.get(self.monitor)
 
         error_msg = (
-            f"Early stopping conditioned on metric `{self.monitor}` which is not available."
+            f"Early stopping conditioned on metric `{self.monitor}` is not available."
```

Comment:
I don't think this change is correct. We are not trying to say that "early stopping [...] is not available". We are pointing to the metric not being available. Was there a misunderstanding?

Reply:
OMG. Yeah, you are right. I hurried before looking into it. 🥲 
I apologize for my silly PR.

Reply:
No problem at all!